amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/LeaseCoordinator.java

```diff
@@ -124,6 +124,11 @@ boolean updateLease(Lease lease, UUID concurrencyToken, String operation, String
      */
     List<ShardInfo> getCurrentAssignments();
 
+    /**
+     * @return All leases
+     */
+    Collection<Lease> getAllAssignments();
```

Comment:
Could you rename the method to `allLeases` in both the interfaces and return a list. Could you also implement a default method to the newly added methods, since they are added to publicly exposed interfaces.

```java
default List<Lease> allLeases() {
    return Collection.emptyList();
}
```

Could you also add javadoc for the methods

Reply:
Ok